v1/fleet_units/logrotate.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Log Rotator for docker containers
+After=docker.service
+
+[Service]
+Restart=always
+ExecStartPre=-/usr/bin/docker stop logrotate
+ExecStartPre=-/usr/bin/docker rm -f logrotate
+ExecStartPre=/usr/bin/docker pull michaloo/logrotate
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill logrotate
+ExecStartPre=-/usr/bin/docker rm logrotate
+ExecStart=/usr/bin/docker run --name logrotate -v /var/lib/docker:/var/lib/docker michaloo/logrotate
```

Comment:
Consider "forking" (or whatever the dockerhub equivalent is) so we own the container and control versioning?